### ISSUE TITLE
FFmpeg 7.1 "Péter" Support

### ISSUE THREAD

**Issue by cyrax13** [jrottenberg/ffmpeg#406]
https://www.ffmpeg.org/download.html#release_7.1

**Comment by jrottenberg** [comment]
Yep thanks, I'm waiting on https://github.com/endoflife-date/endoflife.date/pull/5923 , since I'm rely on that service to get the maintained versions, for each major release.

I might regret it a bit now ;-)

**Comment by chenrui333** [comment]
Should be good now, just ship the endoflife change

**Comment by jrottenberg** [comment]
Awesome, thank you ! 
And for endoflife.date, It makes managing multiple projects a breeze!

**Comment by jsheffie** [comment]
this PR: #407 adds 7.1 support for Ubuntu 24.04

**Comment by jsheffie** [comment]
@jrottenberg Recommend close as 7.1 support was added here: 
- #408

**Comment by cyrax13** [comment]
Solved, closed)
